### PR TITLE
docs: Fix duplicate message and format of 1.17.2 release note and sync it with related security update to zh

### PR DESCRIPTION
**Pull request by wilsonwu**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/news/releases/1.17.x/announcing-1.17.2/index.md
Comment (ericvn) [inline content/en/news/releases/1.17.x/announcing-1.17.2/index.md:50]:
```suggestion
  ([Issue #43371](https://github.com/istio/istio/issues/43371))
```

File: content/en/news/releases/1.17.x/announcing-1.17.2/index.md
Comment (ericvn) [inline content/en/news/releases/1.17.x/announcing-1.17.2/index.md:69]: Point to original PR: https://github.com/istio/istio/pull/44012

File: content/en/news/releases/1.17.x/announcing-1.17.2/index.md
Comment (ericvn) [inline content/en/news/releases/1.17.x/announcing-1.17.2/index.md:68]: This issue does not seem related at all to this particular text. Do we need another bullet?

Comment (ericvn) [reply content/en/news/releases/1.17.x/announcing-1.17.2/index.md:68]: Text in 32210 release note:
```
**Fixed** reconciliation logic in the validation webhook controller to rate-limit
  the retries in the loop. This should drastically reduce churn (and generated logs)
  in cases of misconfiguration.
```

File: content/zh/news/releases/1.17.x/announcing-1.17.2/index.md
Comment (windsonsea) [inline content/zh/news/releases/1.17.x/announcing-1.17.2/index.md:19]:
```suggestion
  (CVSS Score 8.2, High)：客户端可能会伪造 `x-envoy-original-path` 标头信息。
```

File: content/en/news/releases/1.17.x/announcing-1.17.2/index.md
Comment (wilsonwu) [reply content/en/news/releases/1.17.x/announcing-1.17.2/index.md]: Done

File: content/zh/news/security/istio-security-2023-001/index.md
Comment (windsonsea) [inline content/zh/news/security/istio-security-2023-001/index.md:21]:
```suggestion
  (CVSS Score 8.2, High)：客户端可能会伪造 `x-envoy-original-path` 标头信息。
```

File: content/en/news/releases/1.17.x/announcing-1.17.2/index.md
Comment (wilsonwu) [reply content/en/news/releases/1.17.x/announcing-1.17.2/index.md]: Added, will sync the zh version soon.

Comment (wilsonwu) [reply content/en/news/releases/1.17.x/announcing-1.17.2/index.md:68]: All Done

**Comment by ericvn** [comment]
LGTM, but I can't approve until an @istio/wg-docs-maintainers-chinese does, as my approval would also cover that and I don't want the PR to merge without their approval.

**Review by windsonsea — APPROVED** [review]
LGTM for the zh part, thanks.

**Review by ericvn — APPROVED** [review]
Thanks.

**Pull request merged**
